configs/cms-xcache/config.d/52-cms-xcache-lcmaps-authz.cfg

```diff
@@ -0,0 +1,7 @@
+sec.protocol /usr/lib64 gsi -certdir:/etc/grid-security/certificates -cert:/etc/grid-security/xrd/xrdcert.pem -key:/etc/grid-security/\
+xrd/xrdkey.pem -crl:1 -authzfun:libXrdLcmaps.so -authzfunparms:--lcmapscfg,/etc/xrootd/lcmaps.db,--loglevel,5 -gmapopt:10 -gmapto:0
+acc.authdb /etc/xrootd/Authfile-auth
+sec.protbind * gsi
+ofs.authorize 1
+acc.audit deny grant
+xrootd.seclib /usr/lib64/libXrdSec.so
```

Comment:
Is there a reason why this is a separate file from `50-cms-xcache-authz.cfg`? Also, can we just use the same layout/comments as https://github.com/opensciencegrid/xcache/blob/master/configs/stash-cache/config.d/50-stash-cache-authz.cfg where relevant?